engine/version/old_records_cleaner.cpp

```diff
@@ -143,44 +166,44 @@ void OldRecordsCleaner::TryCleanCachedOldRecords(size_t num_limit_clean) {
               "call KVEngine::handleThreadLocalPendingFreeRecords in a "
               "un-initialized access thread");
   auto& tc = cleaner_thread_cache_[access_thread.id];
-  if (tc.old_data_records.size() > 0 || tc.old_delete_records.size() > 0) {
-    maybeUpdateOldestSnapshot();
-    std::unique_lock<SpinMutex> ul(tc.old_records_lock);
-    for (int limit = num_limit_clean;
-         tc.old_delete_records.size() > 0 &&
-         tc.old_delete_records.front().release_time <
-             clean_all_data_record_ts_ &&
-         limit > 0;
-         limit--) {
-      // To avoid access invalid data, an old delete record can be freed only if
-      // no holding snapshot is older than its purging time
-      tc.pending_free_space_entries.emplace_back(PendingFreeSpaceEntry{
-          purgeOldDeleteRecord(tc.old_delete_records.front()),
-          kv_engine_->version_controller_.GetCurrentTimestamp()});
-      tc.old_delete_records.pop_front();
-    }
+  if (tc.old_data_records.empty() && tc.old_delete_records.empty()) {
```

Comment:
Why modify this

Reply:
Reduce two columns of indent to increase readability.